### PR TITLE
Add note about playback policies.

### PR DESCRIPTION
**Pull request by markafoltz**
Addresses Issue #118 by adding an implementation note.  I kept it brief, but could spell out specific scenarios if you think that's helpful.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/remote-playback/pull/131.html" title="Last updated on Jan 24, 2020, 12:29 AM UTC (8d92302)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/remote-playback/131/03e31e4...8d92302.html" title="Last updated on Jan 24, 2020, 12:29 AM UTC (8d92302)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1283,6 +1285,12 @@ <h4>
             connecting</a> and then follow the steps to <a>establish a connection
             with the remote playback device</a>.
           </p>
+          <div class="note">
+            A user agent that implements browser initiated remote playback
+            should consider how it interacts with other browser policies that
+            affect media playback, such as <code><a>autoplay</a></code> policies
```
Comment (mounirlamouri) [inline index.html]: I would avoid linking to HTML's autoplay. If you want to link to something maybe https://html.spec.whatwg.org/multipage/media.html#allowed-to-play would be closer to "autoplay policies"?

Comment (markafoltz) [reply index.html]: Updated the text and link to refer to that definition.

Comment (mounirlamouri) [reply index.html]: I can't see the update

Comment (markafoltz) [reply index.html]: @mounirlamouri  Sorry, forgot to add commit.  PTAL

**Review by mounirlamouri — APPROVED** [review]

**Comment by mounirlamouri** [comment]
Thank you @mfoltzgoogle :)

**Pull request merged**